HospitalFrontend/src/views/scheduling/recommendation/ChooseRecommended.vue

```diff
@@ -53,15 +57,22 @@ export default {
         },
         formAppointmentString:function(appointment){
             
-            //TODO: format dropdown list item
-            /*
-            let appointmentStart=moment(timeInterval.start)
-            let appointmentEnd=moment(timeInterval.end)
             
-            return appointmentStart.format("dddd, MMMM Do YYYY") +
-             " "+appointmentStart.format("h:mm")+" - "+appointmentEnd.format("h:mm")
-            */
+            let appointmentStart=moment(appointment.timeInterval.start)
+            let appointmentEnd=moment(appointment.timeInterval.end)
+            
+            return "Doktor: " + appointment.doctor.person.name + " " + appointment.doctor.person.surname + " - "
+             +appointmentStart.format("dddd, MMMM Do YYYY") +
```

Comment:
Is this string on english or on serbian?

Reply:
Changed to english on all occurrences.